### PR TITLE
Set debug loglevel on qrserver to get the expected log messges. MERGEOK

### PR DESCRIPTION
**Pull request by aressem**

### CODE REVIEW

**Comment by aressem** [comment]
FYI, @gjoranv

File: tests/container/classloading_in_deconstruct/classloading_in_deconstruct.rb [hunk 1]
```diff
@@ -26,11 +26,16 @@ def test_classloading_in_deconstruct
     compile_bundles(@vespa.nodeproxies.values.first)
 
     start(original_application, :bundles => [exporter, importer])
+
+    qrserver = @vespa.nodeproxies.values.first
+    qrserver.execute("vespa-logctl -c qrserver debug=on", :exceptiononfailure => false)
```
Comment (jonmv) [inline tests/container/classloading_in_deconstruct/classloading_in_deconstruct.rb]: Seems to be an `vespa.adminserver.logctl(...)` that is widely used elsewhere?

**Comment by gjoranv** [comment]
debug logging should only be enabled for package com.yahoo.container.jdisc.component

File: tests/container/classloading_in_deconstruct/classloading_in_deconstruct.rb [hunk 2]
```diff
@@ -26,11 +26,16 @@ def test_classloading_in_deconstruct
     compile_bundles(@vespa.nodeproxies.values.first)
 
     start(original_application, :bundles => [exporter, importer])
+
+    qrserver = @vespa.nodeproxies.values.first
```
Comment (jonmv) [inline tests/container/classloading_in_deconstruct/classloading_in_deconstruct.rb]:
```suggestion
```

Comment (gjoranv) [inline tests/container/classloading_in_deconstruct/classloading_in_deconstruct.rb]: this is not necessary in a ContainerTest, just use `@container.logctl()`

File: tests/container/classloading_in_deconstruct/classloading_in_deconstruct.rb (hunk 1 above)
Comment (jonmv) [inline tests/container/classloading_in_deconstruct/classloading_in_deconstruct.rb]:
```suggestion
    vespa.adminserver.logctl("qrserver:com.yahoo.container.jdisc.component", "debug=on")
```

**Comment by jonmv** [comment]
Urgh, suggestions borked once change.

File: tests/container/classloading_in_deconstruct/classloading_in_deconstruct.rb [hunk 3]
```diff
@@ -26,11 +26,15 @@ def test_classloading_in_deconstruct
     compile_bundles(@vespa.nodeproxies.values.first)
 
     start(original_application, :bundles => [exporter, importer])
+
+    vespa.adminserver.logctl("qrserver:com.yahoo.container.jdisc.component", "debug=on")
```
Comment (jonmv) [inline tests/container/classloading_in_deconstruct/classloading_in_deconstruct.rb]:
```suggestion
    @container.logctl("qrserver:com.yahoo.container.jdisc.component", "debug=on")
```

Comment (gjoranv) [inline tests/container/classloading_in_deconstruct/classloading_in_deconstruct.rb]:
```suggestion
        @container.logctl('container:com.yahoo.container.jdisc.component', 'debug=on')
```

**Comment by jonmv** [comment]
PTAL.

**Comment by aressem** [comment]
Did you run the test ?

**Comment by jonmv** [comment]
@gjoranv is.

**Comment by aressem** [comment]
I have verified that this works locally now. But, I would appreciate to test the required suggested changes before my PR is merged.

**Comment by gjoranv** [comment]
I tested it before merging, but I'll let you merge yourself next time :)

**Pull request merged**